kettle3D-versions.py

```diff
@@ -22,122 +22,134 @@
 from os.path import normpath
 from os import getenv
 from os import getcwd
+
+import os
 import time
 import sys
 import pickle
- 
-osnamefile = normpath(getcwd() + "/osname.txt")
+import pathlib
+
+# I've declared directory outside of the `if` block so it can be accessed in the global scope
+directory = ''
 
-osname = open(osnamefile).read()
+# In Python, os.name is 'nt' if you are running on Windows
+if os.name == 'nt': # do windows-specific things

```

Comment:
`osname` is a variable installed by the Kettle3D installer, which either reads 'windows', 'os x', 'linux', or 'unix'. I installed it, so it says 'windows' on 64-bit windows devices.

Reply:
I'm not referencing `os.name`